blockchyp/blockchyp_client.py

```diff
@@ -101,49 +108,85 @@ def is_terminal_routed(self, request):
     def __gateway_get(self, path):
         url = self.gateway_host + "/api" + path
         gateway_config = self.__get_gateway_config()
-        return requests.get(url, gateway_config)
+        reply = requests.get(url, headers=gateway_config)
+
+        return reply
 
     def __get_gateway_config(self):
```

Comment:
As far as I can tell, this method is just a wrapper around crypto.generate_gateway_headers and can go away.